content/helm-nginx/installnginx.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Install bitnami/nginx"
+date: 2018-08-07T08:30:11-07:00
+weight: 40
+---
+
+Installing the Bitnami standalone NGINX web server Chart involves us using the [helm install](https://docs.helm.sh/helm/#helm-install) command.
+
+When we install using Helm, we need to provide a deployment name, or a random one will be assigned to the deployment automatically.
+
+Let's provide **mywebserver** as the deployment name as we install NGINX:
+
+```
+helm install --name mywebserver bitnami/nginx
+```
+Upon running this command, you should see some confirmations around the objects that were created:
+
+```
+NAME:   mywebserver
+LAST DEPLOYED: Tue Nov 13 19:55:25 2018
+NAMESPACE: default
+STATUS: DEPLOYED
+
+RESOURCES:
+==> v1beta1/Deployment
+NAME               AGE
+mywebserver-nginx  0s
+
+==> v1/Pod(related)
+
+NAME                                READY  STATUS             RESTARTS  AGE
+mywebserver-nginx-85985c8466-tczst  0/1    ContainerCreating  0         0s
+
+==> v1/Service
```

Comment:
I would also point to helm recommendations for things to check while it creates resources. You can ignore this if it doesn't apply to nginx chart

Reply:
there don't appear to be any